share/spice/cocoapods/cocoapods.js

```diff
@@ -0,0 +1,51 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_cocoapods = function(api_result){
+
+        // Validate the response (customize for your Spice)
+        if (!api_result || api_result.error) {
+            return Spice.failed('cocoapods');
+        }
+        
+        // Get the original query.
+        var script = $('[src*="/js/spice/cocoapods/"]')[0];
```

Comment:
This method should only be used when necessary. You can opt for `DDG.get_query().replace("cocoapods", "");`